### PR TITLE
Add environment configuration to install tarball

### PR DESCRIPTION
**Pull request by acomodi**
This PR also enables the 200T device installation within the other device's installation, so to have all of the devices in one package only.

### CODE REVIEW

**Comment by kgugala** [comment]
Do we want to restore a200t installation together with the rest of the xc7? I thought we will rather split all the architectures into separate packages.

**Comment by acomodi** [comment]
Makes sense to split all different architectures in sub-packages. That is IMO the best way to proceed. For the time being I wanted to see how much time/size requires to have the 200T included in the same package.

Probably we need to modify also the STPM and the GH actions in https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1719 to get the latest devices and generate one file with a link for each different device. So that in the end we would have the following:

- a link to the binary toolchain (this could potentially end up in a conda package and it will include all the bash scripts to run the toolchain)
- a set of links to the various devices tarballs

File: xc/xc7/tests/install_test/CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,6 @@
 set(INSTALLATION_DIR_BIN "${CMAKE_INSTALL_PREFIX}/bin")
 
-if(${INSTALL_DEVICE} STREQUAL "xc7a200t")
-add_test(NAME binary_toolchain_test
+add_test(NAME binary_toolchain_test_200t
```
Comment (mithro) [inline xc/xc7/tests/install_test/CMakeLists.txt:3]: Is there a `binary_toolchain_test_100t`? Should this also be `xc7a200t`?

**Comment by mithro** [comment]
I think we should probably have an individual package per part, or a combined package which has all the parts. Having one package with `a50t` + `a100t` and another with `a200t`.

File: xc/xc7/tests/install_test/Makefile [hunk 2]
```diff
@@ -17,17 +16,13 @@ ${BUILDDIR}:
 	mkdir ${BUILDDIR}
 
 ${BUILDDIR}/${TOP}.eblif: | ${BUILDDIR}
-ifeq (,$(wildcard ${XDC}))
```
Comment (litghost) [inline xc/xc7/tests/install_test/Makefile:20]: Why is the XDC stuff gone?

**Comment by litghost** [comment]
We should probably have a tarball for each part class (e.g. 50T, 100T, 200T).

File: xc/xc7/tests/install_test/Makefile (hunk 2 above)
Comment (acomodi) [reply xc/xc7/tests/install_test/Makefile:20]: I changed the test to use XDC only

**Comment by mithro** [comment]
> We should probably have a tarball for each part class (e.g. 50T, 100T, 200T).
Agreed!

My sentence should have ended with the following;
> I think we should probably have an individual package per part, or a combined package which has all the parts. Having one package with `a50t` + `a100t` and another with `a200t` **is weird**.

**Review by HackerFoo — APPROVED** [review]

**Comment by acomodi** [comment]
This PR seems to break the install CI. Unsure what is the reason as the log is empty. Maybe it takes too much time/memory/disk space?

**Comment by acomodi** [comment]
I have modified this PR to only add the environment file to the tarball. The previous changes in this PR have been moved [here](https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1747) alongside with a restructure of the install packages generated by CI

**Pull request merged**